### ISSUE TITLE
ScaleMode Added to VideoPlayer

### ISSUE THREAD

**Issue by squidfingers** [squidfingers/starkplayer#6]
A new parameter called 'scalemode' has been added to the video player. The acceptable values are 'letterbox' and 'stretch'. If the parameter is not set, 'letterbox' will be used. Feel free to add this parameter to the jquery plugin.

In the future, I might add 'zoom' as an acceptable value, but that aspect ratio causes a bit of trouble in fullscreen mode. I need to think about it a bit more.

Also, Adobe uses the values 'maintainAspectRatio' and 'exactFit' instead of 'letterbox' and 'stretch'. I feel that Adobe's values are too wordy, although I'm not in love with 'letterbox'. Can you think of something better?

**Comment by alanchrt** [comment]
The property is now called 'aspectratio' with values of 'maintain' or 'stretch.'
